docs/nx-cloud/features/github-integration.md

```diff
@@ -0,0 +1,53 @@
+# GitHub integration
+
+Any CI tool requires tight integration with your existing version control system. Nx Cloud offers first class integration with GitHub in the following ways.
+
+## Easy Onboarding
+
+![Screenshot of Nx Cloud connecting a GitHub repository](/nx-cloud/features/github-onboarding.avif)
+
+Ready to get started with Nx Cloud? Get started with Nx Cloud in no time with our GitHub connection process. Connect your workspace, and Nx Cloud will create a pull request with everything you need. Select your workspace and organization, and Nx Cloud takes care of the rest. User access is automatically connected to GitHub, and a PR is created to connect your workspace. Your repo now has [distributed caching](/ci/features/remote-cache) in less than 5 minutes.
+
+## Pull Request insights
+
+![Screenshot of Nx Cloud app for GitHub showing pull request insights](/nx-cloud/features/github-pr-bot.avif)
+
+Good CI checks require fast and easy access to results. That's why Nx Cloud will update your PR with the current running status of your tasks and a convenient link to your Nx Cloud results and logs. Take advantage of the enhanced developer experience of structured and searchable logs. Quick insight to PR task progress, so you're not stuck waiting for every task to complete. And with Nx Replay, developers can quickly replay tasks locally to avoid running tasks that CI has already completed.
+
+## Access control
+
+![Diagram showing users syncing from GitHub to Nx Cloud](/nx-cloud/features/github-user-management.avif)
+
+Nx Cloud organizations can use their existing GitHub access controls to manage Nx Cloud as well. This allows Nx Cloud to fit in to any existing on-boarding or off-boarding process. There's no need to manually manage users separately. Get your engineers Nx Cloud access right alongside their GitHub access so they can get to work fast. Use [personal access tokens](/ci/recipes/security/personal-access-tokens) to further enhance your security.
+
+## Getting started
+
+First, you'll want to connect your Nx Cloud account to GitHub. You can use your regular username and password or log in via Google or GitHub, connecting to GitHub is a separate step.
+
+{% call-to-action title="Log in to Nx Cloud" url="https://cloud.nx.app/profile/vcs-integrations" icon="nxcloud" description="Connect to GitHub via your profile settings" %}
+Connect to GitHub
+{% /call-to-action %}
```

Comment:
This button's primary text is "Log in to Nx Cloud", but that isn't the main thing you're trying to get people to do.  Also, "Connect to GitHub" isn't displayed anywhere.  Maybe that should be the title?

Reply:
Thanks, both CTAs have been updated with (hopefully) better copy